### PR TITLE
Replaced replication controllers with deployments

### PR DESCRIPTION
**Pull request by glnds**
Fully tested against Google Cloud.

### CODE REVIEW

**Comment by glnds** [comment]
I signed it!

File: docs/getting-started-guides/meanstack.md [hunk 1]
```diff
@@ -20,8 +20,8 @@ Before we jump in and start kube’ing it up, it’s important to understand som
...
-* Services: This is the other side to Replication Controllers. A service is the single point of contact for a group of Pods. For example, let’s say you have a Replication Controller that creates four copies of a web server pod. A Service will split the traffic to each of the four copies. Services are "permanent" while the pods behind them can come and go, so it’s a good idea to use Services.
+* Deployments: A Deployment provides declarative updates for Pods and ReplicaSets. You only need to describe the desired state in a Deployment object, and the deployment controller will change the actual state to the desired state at a controlled rate for you. You can define Deployments to create new resources, or replace existing ones by new ones.
+* Services: This is the other side to Deployments. A service is the single point of contact for a group of Pods. For example, let’s say you have a Deployment that creates four copies of a web server pod. A Service will split the traffic to each of the four copies. Services are "permanent" while the pods behind them can come and go, so it’s a good idea to use Services.
```
Comment (johndmulhausen) [inline docs/getting-started-guides/meanstack.md]: Remove sentence: "This is the other side to Deployments." 

Subjective/distracting/weird.

File: docs/getting-started-guides/meanstack.md [hunk 2]
```diff
@@ -20,8 +20,8 @@ Before we jump in and start kube’ing it up, it’s important to understand som
...
-* Replication Controllers: This is the management component of Kubernetes, and it’s pretty cool. You give it a set of Pods, tell it "I want three copies of this," and it creates those copies on your cluster. It will do its best to keep those copies always running, so if one crashes it will start another.
-* Services: This is the other side to Replication Controllers. A service is the single point of contact for a group of Pods. For example, let’s say you have a Replication Controller that creates four copies of a web server pod. A Service will split the traffic to each of the four copies. Services are "permanent" while the pods behind them can come and go, so it’s a good idea to use Services.
+* Deployments: A Deployment provides declarative updates for Pods and ReplicaSets. You only need to describe the desired state in a Deployment object, and the deployment controller will change the actual state to the desired state at a controlled rate for you. You can define Deployments to create new resources, or replace existing ones by new ones.
```
Comment (janetkuo) [inline docs/getting-started-guides/meanstack.md]: `ReplicaSet` isn't used / explained anywhere in this doc. Consider removing it. Also, suggest rephrasing the description of Deployments more about Pods, e.g. 

* Deployments: A Deployment provides declarative updates for Pods. You can define Deployments to create new Pods, or to replace existing Pods by new ones.

**Comment by janetkuo** [comment]
@glnds only one comment, otherwise this is good to go. Thanks!

**Comment by pwittrock** [comment]
@glnds What is the status of this PR?  Should I close it?

**Comment by glnds** [comment]
@pwittrock Sorry this slipped my mind for a while. Changed the definition as suggested. Fine for me now.

**Pull request merged**